spec/solver.spec.rb

```diff
@@ -0,0 +1,43 @@
+require_relative '../solver'
+
```

Comment:
![Screen Shot 2022-09-01 at 1 57 57 PM](https://user-images.githubusercontent.com/83717285/187992095-d590516f-c255-44c9-81e1-11c03137110d.png)

- [OPTIONAL] Your tests are set up correctly, great work! I would recommend using different names for the tests instead of the same name for everyone, e.g: `should return 120 from factorial(5)`, this way is easier to read the test results.